### DISCUSSION TITLE
I cannot create other accouts rather than the main email address.

### DISCUSSION THREAD

**Discussion by JoseArias31** [supabase/supabase#30447] [Questions] [answered]
I have successfully coded my registration page. I can create an account with my email address. However, when I try to create another account with another email address, I get the message "This email address is not authorized." That's strange. I'd appreciate any help.

**Comment by GaryAustin1** [comment, marked as answer]
You need to have your own SMTP provider.  The built in one only works for dashboard emails.
![image](https://github.com/user-attachments/assets/0eb531c5-fd78-4522-80fb-3885ba73c737)
https://supabase.com/docs/guides/auth/auth-smtp

**Reply by JoseArias31** [reply]
Awesome! Thanks a lot, I was considering an error code on my end, but I need to set it up a SMPT provider first.
